assets/src/edit-story/components/helpCenter/navigator/index.js

```diff
@@ -38,12 +42,23 @@ export const NavigationWrapper = styled.div`
   position: absolute;
   left: 0;
   bottom: 0;
-  width: ${NAVIGATION_WIDTH}px;
+  max-height: calc(100vh - ${DISTANCE_FROM_TOP + DISTANCE_FROM_BOTTOM}px);
+  width: ${NAVIGATION_WIDTH + 2}px; /* account for border width */
   color: ${({ theme }) => theme.colors.fg.primary};
   background-color: ${({ theme }) => theme.colors.bg.primary};
   border: 1px solid ${({ theme }) => theme.colors.bg.tertiary};
   border-radius: ${({ theme }) => theme.borders.radius.small};
-  overflow: ${({ isOpen }) => (isOpen ? 'visible' : 'hidden')};
+  overflow: hidden;
+
+  ${({ isOpen }) =>
+    !isOpen &&
+    css`
+      &,
+      * {
+        height: 0;
+        visibility: hidden;
+      }
+    `}
 `;
```

Comment:
Doesn't keep content rendered but invisible on the screen. Hides it so that the user can interact with other elements.

Reply:
Also fixes the reading for screen readers:

|Before|After|
|--|--|
|![broken-lol](https://user-images.githubusercontent.com/22185279/124329311-79a64480-db48-11eb-8a44-9393fbf40a6e.gif)|![fixed-yay](https://user-images.githubusercontent.com/22185279/124329325-7ca13500-db48-11eb-8734-4450a8c60f33.gif)|